reframe/core/schedulers/local.py

```diff
@@ -182,5 +182,5 @@ class _LocalNode(sched.Node):
     def __init__(self, name):
         self._name = name
 
-    def is_available(self):
-        return True
+    def in_state(self, state):
+        return NotImplemented
```

Comment:
This will break the local scheduler. The idea behind returning `True` previously was that the flex alloc nodes would work with `state=idle` for the local scheduler as well. I think we had a PR specifically addressing this in the past.

Reply:
@vkarak Yes it was related to issue https://github.com/eth-cscs/reframe/issues/1087. Do you think that I should return True for idle and False otherwise?

Reply:
Yes, please do that.